sdk/core/azure-core-experimental/azure/core/experimental/transport/_pyodide.py

```diff
@@ -85,9 +104,13 @@ def __init__(self, pipeline: Pipeline, response: PyodideTransportResponse, *_, *
         self.done = False
 
     async def __anext__(self) -> bytes:
-        """Get the next block of bytes."""
+        """Get the next block of bytes.
+
+        :return: The next block of bytes.
+        :rtype: bytes
+        """
         if self._closed:
-            raise StopAsyncIteration()
+            raise StopAsyncIteration()  # pylint: disable=raise-missing-from
```

Comment:
```suggestion
            raise StopAsyncIteration() from None
```

Reply:
I think its an issue with the line spacing -- error thrown for line 166, this can be removed

Reply:
My error, I was fixing stuff without re-running pylint.